cli/src/stake.rs

```diff
@@ -255,24 +255,24 @@ impl StakeSubCommands for App<'_, '_> {
         )
         .subcommand(
             SubCommand::with_name("split-stake")
-                .about("Split a stake account")
+                .about("Create a new stake account using tokens from an existing one")
```

Comment:
Wonder if this should be more explicit. This sounds like it creates a new "clean" stake account, but in fact the the state of the original account (delegated, stake warmed up, lockup, etc) is duplicated.
I think the word "split" is better than "create" in that regard.